client/src/pages/addFlight/index.js

```diff
@@ -9,20 +9,49 @@ import {
   Header,
   SideNav
 } from "../../components";
+
 import "./addflight.css";
 
-class UpdateFlight extends Component {
+class AddFlight extends Component {
   state = {
-    selectValue: ""
+    form: {},
+    originValue: "",
+    destinationValue: ""
   };
 
   handleAddFlight = e => {
     e.preventDefault();
-    console.log("on added flights will be handled here");
+    const {
+      originValue,
+      destinationValue,
+      dateValue,
+      airCraftTypeValue
+    } = this.state.form;
+    const data = JSON.stringify({
+      originValue,
+      destinationValue,
+      dateValue,
+      airCraftTypeValue
+    });
+    fetch("/addFlight_post", {
+      credentials: "same-origin",
+      headers: {
+        "content-type": "application/json"
+      },
+      method: "POST",
+      body: data
+    });
```

Comment:
What do u do with the data returned by the fetch?

Reply:
@sallamTanna Nevermind about this one, I missed the `POST` sign.

Reply:
Focus on names please hehehehhe @NoureldeanSaed

Reply:
:unamused: